importfix/shapely/_geometry_helpers.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
Here we should have python or python3?

Reply:
Depends; most distros today have python symlinked to python3 (as IMO one should, py2 is very deprecated), with scarce few outliers remaining.
On unikraft however it is irrelevant since the hashbang line is never taken into account, and even if it were we have python -> python3 in the standard rootfs.

Reply:
Gotcha, thanks for clarification!